queue_job_subscribe/tests/test_job_subscribe.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2016 Cédric Pigeon
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import odoo.tests.common as common
+
+from odoo.addons.queue_job.job import Job
+
+
+class TestJobSubscribe(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
+        grp_queue_job_manager = self.ref("queue_job.group_queue_job_manager")
```

Comment:
can you please disable tracking and use no_reset_password when creating users? (or use `odoo.tests.common.new_test_user`.
To be done in a separated commit that can be backported.